src/pyfaf/common.py

```diff
@@ -224,6 +225,39 @@ def get_temp_dir(subdir=None):
     return os.path.join(basetmp, userdir, subdir)
 
 
+def _get_config_or_env(conf_s, env_s, default):
```

Comment:
I think the order should be switched. First check the env variables, then the variables from config file.